CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
+## v6.0.1 (November 29, 2021)
```

Comment:
I don't think we'll get this published today; do you want to make a master PR?

Reply:
Shall I update with today's date v6.0.1 (November 30, 2021) ?

Reply:
Yep we can try to get this published today but more realistically tomorrow.

Reply:
Sure let me update it to v6.0.1 (December  1, 2021)